calculation/gmhazard_calc/gmhazard_calc/verification/GMM/psha_im_rrup_plot.py

```diff
@@ -161,12 +162,15 @@ def get_computed_gmms(
                                         )
                                 else:
                                     result = empirical_factory.compute_gmm(
-                                        fault,
-                                        site,
-                                        empirical_factory.GMM[model],
-                                        im,
+                                        fault, site, empirical_factory.GMM[model], im,
                                     )
                                     # result is always tuple
+                                    # For Meta
+                                    # result_dict[tect_type][im][vs30][mag][model].append(
+                                    #     result[0][0]
+                                    #     if isinstance(result, Iterable)
+                                    #     else result[0]
+                                    # )
```

Comment:
Yup, related to the comment above. 

If we use the META config - which contains multiple models, I noticed some cases with List and some cases with Tuple.
Hence, I chose to use `Iterable`

And to avoid confusion, replacing the location of the comment
`# result is always tuple`

Reply:
As an explanation to Joel:

It's a bit gross having comments like this but because it's a quick script, rather than an integrated part, it's kinda easy to have options as comments. Rather than a full argparse which makes it a bit harder to make modifications on the fly to this script.